### PR TITLE
 Check Documentation

### PR DESCRIPTION
**Pull request by Bharadwajshivam28**
Changed the link under  Check Documentation

### CODE REVIEW

File: src/pages/ProjectsPage.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const ProjectsPage = () => {
       <p className="mt-3 text-gray-300 text-[1.2rem] text-center mx-auto w-10/12">
         Want to add your projects?
         <a
-          href="https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md"
+          href="https://projectshut.vercel.app/AddYourProjectsGuide"
           target={"_blank"}
```
Comment (priyankarpal) [inline src/pages/ProjectsPage.jsx:34]: just remove `target={"_blank"}`
```suggestion
        
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Bharadwajshivam28** [comment]
okay

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by Bharadwajshivam28** [comment]
> Looks Good tada

thanks

**Pull request merged**